### PR TITLE
landing page done

### PR DESCRIPTION
**Pull request by MarquesJ8023**

### CODE REVIEW

**Review by ebcitron — COMMENTED** [review]
Looks really good! I am genuinly impressed

File: components/dropdown/dropdown.less [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.dropdown {
```
Comment (ebcitron) [inline components/dropdown/dropdown.less:1]: This ends up looking really nice marques

File: components/tabs/tabs.less [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    z-index         : 2;
+    border-right    : 1px solid @dg;
+    background-color: white;
+    color           : black;
+
+    &:hover {
+      background-color: white;
+    }
+
+    @media @mobile {
```
Comment (ebcitron) [inline components/tabs/tabs.less:76]: Finish up the break points and your almost done!

**Review by ebcitron — APPROVED** [review]

**Comment by MarquesJ8023** [comment]
It's simple, bro. Compared to what everybody else is doing.

**Pull request merged**